cli/src/pcluster/constants.py

```diff
@@ -325,3 +325,5 @@ class Operation(Enum):
 IAM_ROLE_REGEX = "^arn:.*:role/"
 IAM_INSTANCE_PROFILE_REGEX = "^arn:.*:instance-profile/"
 IAM_POLICY_REGEX = "^arn:.*:policy/"
+
+EXTERNAL_SLURMDBD_ASG_SIZE = "1"
```

Comment:
I don't think this constant should be part of the PC codebase.

Reply:
Have moved!